docs/user-roles.rst

```diff
@@ -44,3 +53,58 @@ also perform every available operation inside that project.
     organization.
 
 A *project member* has read-only access to the project.
+
+
+.. _user-ops:
+
+User operations
+===============
+
+
+.. _system-user:
+
+``System`` user
+---------------
+
+In CrateDB Cloud, the backend uses a user called ``system`` in order to perform
+Cloud cluster upgrades, backups and scaling functions, among other things.
+
+.. WARNING::
+    The user ``system`` is essential for CrateDB Cloud to function as intended.
+    While it is not normally accessible through the CrateDB Console or the
+    Croud CLI, it can be accessed through the CrateDB Admin UI or any other SQL
+    client. It is important not to edit or delete this user in any way.
+    Otherwise, the functioning of Cloud clusters may be compromised.
+
+
+.. _crate-user:
+
+``Crate`` user
+--------------
+
+In CrateDB, when you create a cluster node (through whatever method), a
+``crate`` user is automatically generated. ``crate`` is a superuser and can
+perform all possible operations. For CrateDB Cloud purposes, however, this user
+is not exposed to the Cloud customer, who instead creates their own users
+according to the user roles and privileges described above. (For information on
+how to do so, see our documentation on `adding users`_).
```

Comment:
I think there's a misunderstanding:
a "CrateDB Cloud" user is different from a "CrateDB" user.
The former is used to interact with the CreatDB Cloud application. That involves creating org/projects, adding/removing other users, deploying clusters, etc..
The "CrateDB" user is merely a user of the database service that is offered on cratedb cloud.

So these two types of users are not connected at all.
This should only be about "CrateDB" users.
The CrateDB Cloud user creates their own "CrateDB" user when they deploy a new cluster (that's what username and password is for on step 2 of the wizard). And it is about this very database user that doesn't have access to the operations below. It has nothing to do with the CrateDB Cloud user.

So when a new cluster is being deployed, the CrateDB instance creates the "Crate" user (automatically from the deployment script) and a second database user (from the information from the wizard) so the customer can access CrateDB with the latter.

Hope that makes things clearer. :)